.circleci/config.yml

```diff
@@ -177,9 +175,11 @@ workflows:
           context:
             - hmpps-common-vars
       - hmpps/npm_security_audit:
+          slack_channel: << pipeline.parameters.alerts-slack-channel >>
           context:
             - hmpps-common-vars
       - hmpps/veracode_policy_scan:
+          slack_channel: << pipeline.parameters.alerts-slack-channel >>
```

Comment:
Moved these to use shared param so its more obvious where the value is coming from rather than using defaults